src/cue.c

```diff
@@ -93,8 +56,8 @@ track_break_write_cue(int index, gboolean write, gulong start_offset, gulong end
 
     gchar *time = track_break_format_timestamp(start_offset, TRUE);
 
-    fprintf(fp, "TRACK %02d AUDIO\n", index + 1);
-    fprintf(fp, "INDEX 01 %s\n", time);
+    fprintf(fp, "\n\tTRACK %02d AUDIO\n", index + 1);
+    fprintf(fp, "\t\tINDEX 01 %s\n", time);
```

Comment:
Isa this intended to be `\t\t` and not `\n\t`?

Reply:
No, this was intentional. It outputs a file like this:

```CUE
REM Generated with wavbreaker
FILE "file.mp3" WAVE

	TRACK 01 AUDIO
		INDEX 01 0:00:00

	TRACK 02 AUDIO
		INDEX 01 0:43:00

	TRACK 03 AUDIO
		INDEX 01 3:01:00

	TRACK 04 AUDIO
		INDEX 01 9:29:00

	TRACK 05 AUDIO
		INDEX 01 12:40:00

	TRACK 06 AUDIO
		INDEX 01 16:38:00

	TRACK 07 AUDIO
		INDEX 01 19:24:00
```